src/governance-object.h

```diff
@@ -25,6 +25,7 @@ class CGovernanceVote;
 
 static const int MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024;
 static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70204;
+static const int GOVERNANCE_FILTER_PROTO_VERSION = 70206;
```

Comment:
See my comment above.  I find it preferable to keep all governance parameters in the same place (it makes it easier to find them).